Resources/doc/basic-usage.rst

```diff
@@ -1,55 +1,148 @@
+
 Basic Usage
 ===========
 
-This bundle works by configuring a set of filters and then applying those
-filters to images inside a template. So, start by creating some sort of filter
-that you need to apply somewhere in your application. For example, suppose
-you want to thumbnail an image to a size of 120x90 pixels:
+Generally, this bundle works by applying filter sets to images from inside
+a template. Your filter sets are defined within the application's configuration
+file (often ``app/config/config.yml``) and are comprised of a collection of
+filters, post-processors, and other optional parameters.
+
+We'll learn more about post-processors and other available parameters later,
+but for now lets focus on how to define a simple filter set comprised of a
+few filters.
+
+
+Create Thumbnails
+-----------------
+
+.. _usage-create-thumbnails:
```

Comment:
In Symfony Docs we usually put the label before the section title. Otherwise, when clicking a link with this anchor, you don't have a clear reference about where you are.